src/main/java/baritone/process/BuilderProcess.java

```diff
@@ -182,9 +181,8 @@ public void buildOpenSchematic() {
 
     @Override
     public void buildOpenLitematic() {
-        if (LitematicaHelper.isLitematicaPresent()) {
-            SchematicPlacementManager placementManager = DataManager.getSchematicPlacementManager();
-            List<SchematicPlacement> placementList = placementManager.getAllSchematicsPlacements();
+        if (LitematicaHelper.isLitematicaPresent()) { //TODO Investigate why true even without litematica being present
+            List<SchematicPlacement> placementList = DataManager.getSchematicPlacementManager().getAllSchematicPlacements();
             if (placementList.size()>0) {
                 String name = LitematicaHelper.getName(placementList,0);
```

Comment:
atm first schematic in the list is built. evaluate options

- leave as is aka first in list gets built
- for loop that builds all loaded schematics
- selection option (0=no schem error, 1= auto select, >1 = ask player)